src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -827,7 +834,7 @@ private DPath GetFunctionNamespace(CallNode node, TexlVisitor visitor)
 
         internal bool TryGetDataQueryOptions(out DataSourceToQueryOptionsMap tabularDataQueryOptionsMap)
         {
-            return TryGetDataQueryOptions(Top, false, out tabularDataQueryOptionsMap);
```

Comment:
>false [](http://example.com/codeflow?start=47&length=5)

why are these params removed?

Reply:
look at line 632 in this file and you'll that that forCodegen is never used, so we can remove it